api/agent/call.go

```diff
@@ -73,6 +73,11 @@ func FromHTTPFnRequest(app *models.App, fn *models.Fn, req *http.Request) CallOp
 			}
 		}
 
+		invokeType := req.Header.Get("Fn-Invoke-Type")
```

Comment:
I don't see any change to `GetCall` in this patch to use `ioutil.Discard` as the writer when we're doing async calls. that change probably won't go in this file, but there should be a `WithWriter(ioutil.Discard)` somewhere I think (in pure runner and lb agent?)? we just want logs (stderr) right ?

Reply:
cat skinning it appears, feel free to ignore/resolve - this would be one reason to get rid of intermediate buffers we have, maybe one day ;)

Reply:
@rdallman I'm not quite sure what you mean here, could you explain further?

Reply:
rather than specifying `ioutil.DIscard`, the pure runner uses a writer that does not write the bytes to anywhere (https://github.com/reclaro/fn/pull/1/files#diff-4a61f8b065f6a7e1b236e138ae1e6b81R407). I have a mild preference to configure in GetCall since it avoids having branch statements all over the actual logic, but the current code works fine at least...